mysql-connector-cpp.yaml

```diff
@@ -0,0 +1,82 @@
+package:
+  name: mysql-connector-cpp
+  version: 9.0.0
+  epoch: 0
+  description: MySQL Connector/C++ is a MySQL database connector for C++
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - openssl-dev
+      - zlib-dev
```

Comment:
Please use as many system packages as possible. I.e. protobuf zlib lz4 zstd. We do not want vendored copies of those; if at all possible to use system ones.

Reply:
Thanks done. Also passed the following flags:
```
-DWITH_BOOST=system \
-DWITH_LZ4=system \
-DWITH_SSL=system \
-DWITH_ZLIB=system \
-DWITH_ZSTD=system
```

But passing `-DWITH_PROTOBUF=system` fails with syntax error due to probuf version, i guess.

```
/home/build/cdk/protocol/mysqlx/protocol.cc:131:17: error: 'log_handler' was not declared in this scope
/home/build/cdk/protocol/mysqlx/protocol.cc:131:3: error: 'SetLogHandler' was not declared in this scope
```

Reply:
sadness, likely everything needs to be regenerated or it is incompatible. Leave it as it is, we have tried =)